Open Source Project models

VeneratioVitae opened the pull request:
First attempt without too much preprocessing and optimization... Few visualizations also

VeneratioVitae committed:
Add files via upload

nuel-d1 reviewed (commented):
Your commit message isn't explicit enough to help one understand what you did in the code. Care to explain further?

Oba-max22 reviewed (approved):
Great work, this is worth merging

VeneratioVitae committed:
Created using Colaboratory
Preprocessing,
Exploratory Data Analysis,
Correlation Analysis,
and Predictive modelling done here...

VeneratioVitae commented:
@nuel-d1 I'm sorry, i don't quite get that. What do you need me to include in my commit messages
